src/Sentry.OpenTelemetry/SentrySpanProcessor.cs

```diff
@@ -113,7 +113,7 @@ public override void OnEnd(Activity data)
         // Make a dictionary of the attributes (aka "tags") for faster lookup when used throughout the processor.
         var attributes = data.TagObjects.ToDict();
 
-        if (attributes.TryGetTypedValue("http.url", out string? url) && (_options?.IsSentryRequest(url) ?? false))
+        if (attributes.TryGetTypedValue("url.full", out string? url) && (_options?.IsSentryRequest(url) ?? false))
```

Comment:
this approach to detect if it's a Sentry request seems rather fragile

Reply:
Other ideas?

Reply:
AsyncLocal?